SkietbaanBE/SkietbaanBE/EmailService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net;
+using System.Net.Mail;
+using System.Linq;
+using SkietbaanBE.Models;
+using System.Net.Mime;
+using System.IO;
+
+namespace SkietbaanBE
+{
+    public class EmailService
+    {
+
+        readonly string smtpAddress = "smtp.gmail.com";
+        readonly int portNumber = 587;
+        readonly bool enableSSL = true;
+        readonly string emailFromAddress = "bathande704@gmail.com"; //Sender Email Address
+        readonly string password = "22842633"; //Sender Password
```

Comment:
Why are your passwords stored like this, I'm sure that has been discussed already

Reply:
wow

Reply:
i was not using that class forgot to delete it